### PR TITLE
Add Apollo Vue Devtools

### PR DESCRIPTION
**Pull request by HelloAlexPan**
- [x] Pull request template structure not broken

### `Type`

> ℹ️  What types of changes does your code introduce?

Add apollo-vue-devtools

> 👉 _Put an `x` in the boxes that apply_

- [ ] Fix
- [ ] Feature 

### `Checklist`

> ℹ️  Check all checkboxes - this will indicate that you have done everything in accordance with the rules in [CONTRIBUTING](https://github.com/vuejs/awesome-vue/blob/master/.github/contributing.md).

> 👉  _Put an `x` in the boxes that apply._

- [x] Title as described
- [x] Make sure you put things in the right category!
- [x] Always add your items to the end of a list

#### `Open Source`

- [x] Link description does not contain a link to an author / third-party resource
- [x] The documentation (README) contains a description of the project, illustration of the project with a demo or screenshots and a CONTRIBUTING section
- [x] The documentation is in English.
- [x] The project is active and maintained.
- [x] The project accepts contributions.
- [x] Not a commercial product

#### `Apps/Websites`

- [ ] The website is available without errors or ssl certificate problems, and load in a reasonable amount of time.
- [ ] The website is using vuejs intensively. It should detect vue with [vue-devtools](https://github.com/vuejs/vue-devtools).
	> See [Storipress Login](https://stori.press/settings/auth/sign-in)
  > If you cannot detect vue with `vue-devtools` due to work at non public pages (e.g. for enterprise website), you can send Pull Request with screenshot that detected it.
- [ ] The website is original and not too simple. For that reason, blogs and simple landing pages are rejected.
- [ ] A commercial product using Vue, provided that guests could reasonably check out how Vue was used (i.e. A headless CMS which uses Vue for the Admin/editor Area and offers a free tier).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3359,6 +3361,7 @@ _Inspecting & debugging_
 - [VueSource](https://github.com/davestewart/vue-source) - Global Vue mixin which identifies components in source code by adding HTML comments
 - [NW-Vue-DevTools](https://github.com/TheJaredWilcurt/nw-vue-devtools) - DevDependency for adding Vue DevTools into NW.js
 - [bruit-io](https://github.com/Moventes/bruit.io) - Collect feedbacks with screenshot and technical data and post them to an API like [bruit.io](https://bruit.io).
+- [apollo-vue-devtools](https://github.com/storipress/apollo-vue-devtools) - Apollo integration for Vue-Devtools to inspect GraphQL queries in the vue-devtools timeline
```
Comment (Scrum) [inline README.md:3364]: you must select one of the categories

File: README.md [hunk 2]
```diff
@@ -8,6 +8,7 @@
 ## Awesome Vue.js [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome) [![Track Awesome List](https://www.trackawesomelist.com/badge.svg)](https://www.trackawesomelist.com/vuejs/awesome-vue)
 
 > A curated list of awesome things related to Vue.js
+- [Awesome Vue.js ![Awesome](https://github.com/sindresorhus/awesome) [![Track Awesome List](https://www.trackawesomelist.com/badge.svg)](https://www.trackawesomelist.com/vuejs/awesome-vue)](#awesome-vuejs--)
```
Comment (Scrum) [inline README.md:11]: ?

**Review by Scrum — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (HelloAlexPan) [reply README.md:11]: My IDE adds this on save — I assumed this was added as you guys had some custom prettier config

File: README.md (hunk 1 above)
Comment (HelloAlexPan) [reply README.md]: Done

**Review by Scrum — APPROVED** [review]

**Pull request merged**